core-bundle/src/Twig/Loader/ContaoFilesystemLoader.php

```diff
@@ -391,10 +391,6 @@ public function buildInheritanceChains(): void
      */
     private function getThemeTemplateName(string $name): ?string
     {
-        if (!\is_string($name)) {
```

Comment:
I forgot to do this (see https://github.com/contao/contao/pull/3343/files#diff-e36795bda9eb0543af8a7afe0e4f71cd1e57f8d12fcf3bb49c8a2cdc8f2df976R402).